editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -49,16 +49,18 @@ const DRAG_START_TRESHOLD = 2
 export function isResizing(editorState: EditorState): boolean {
   const dragState = editorState.canvas.dragState
   return (
-    dragState?.type === 'RESIZE_DRAG_STATE' &&
-    getDragStateDrag(dragState, editorState.canvas.resizeOptions) != null
+    (dragState?.type === 'RESIZE_DRAG_STATE' &&
+      getDragStateDrag(dragState, editorState.canvas.resizeOptions) != null) ||
+    editorState.canvas.interactionState != null
   )
 }
 
 export function isDragging(editorState: EditorState): boolean {
   const dragState = editorState.canvas.dragState
   return (
-    dragState?.type === 'MOVE_DRAG_STATE' &&
-    getDragStateDrag(dragState, editorState.canvas.resizeOptions) != null
+    (dragState?.type === 'MOVE_DRAG_STATE' &&
+      getDragStateDrag(dragState, editorState.canvas.resizeOptions) != null) ||
+    editorState.canvas.interactionState != null
```

Comment:
This will be true in any kind of new interaction (not just dragging), although the function is called `isDragging`

Reply:
I think this is good like this, isDragging is used by some to-be-deleted controls outside the file, _for now_ I think I prefer if we don't refactor that code, and we can just lie to them and say that any new Strategy-related interaction state is a "drag"